kotlin-muix-date-pickers/src/jsMain/generated/muix/pickers/DesktopDatePicker.types.kt

```diff
@@ -0,0 +1,27 @@
+// Automatically generated - do not modify!
+
+package muix.pickers
+
+external interface DesktopDatePickerProps<TDate, TEnableAccessibleFieldDOMStructure> : react.Props {
+    /**
+     * Overridable component slots.
+     * @default {}
+     */
+    var slots: DesktopDatePickerSlots<TDate>?
+
+    /**
+     * The props used for each component slot.
+     * @default {}
+     */
+    var slotProps: DesktopDatePickerSlotProps<TDate, TEnableAccessibleFieldDOMStructure>?
+
+    /**
+     * Years rendered per row.
+     * @default 4
+     */
+    var yearsPerRow: dynamic
```

Comment:
Please use `Any` or `Any?` instead here and in other declarations.
`dynamic` type usage is strictly not recommended as non-multiplatform.